lib/jt400.ts

```diff
@@ -118,7 +119,15 @@ function createInstance(connection, insertListFun, inMemory) {
 
     obj.query = function(sql, params) {
       const jsonParams = paramsToJson(params || [])
-      return Q.nfcall(thisConn.query, sql, jsonParams).then(JSON.parse)
+      return Q.nfcall(thisConn.query, sql, jsonParams)
+        .then(JSON.parse)
+        .catch(error => {
+          throw newProgrammerOops(
+            error.stack.split('\n')[1],
```

Comment:
Can't we use error.message ?

Reply:
error.message returns the whole stack,

> 
> Error running instance method
> java.sql.SQLException: Descriptor index not valid.
> 	at com.ibm.as400.access.JDError.createSQLExceptionSubClass(JDError.java:828)
> 	at com.ibm.as400.access.JDError.throwSQLException(JDError.java:412)
> 	at com.ibm.as400.access.AS400JDBCPreparedStatement.setValue(AS400JDBCPreparedStatement.java:3552)
> 	at com.ibm.as400.access.AS400JDBCPreparedStatement.setString(AS400JDBCPreparedStatement.java:3231)
> 	at nodejt400.JdbcJsonClient.setParams(Unknown Source)
> 	at nodejt400.JdbcJsonClient.query(Unknown Source)
> 	at nodejt400.JT400.query(Unknown Source)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.lang.reflect.Method.invoke(Method.java:498)
> 

I will put "Failed dto execute query"